Gemfile

```diff
@@ -0,0 +1,8 @@
+source 'https://rubygems.org'
+
+gemspec
+
+group :development, :test do
```

Comment:
Recommend moving these to the gemspec (using `s.add_development_dependency`). This file should end at line 3 unless you need to specify the git location of a dependency.